language/move-lang/tests/move_check/parser/byte_string.move

```diff
@@ -0,0 +1,12 @@
+module M {
+    public fun byte_string(): vector<u8> {
+        b"Libra"
+    }
+    public fun byte_string_with_quotes(): vector<u8> {
+         b"\"Libra\""
+    }
+    public fun multi_line_byte_string(): vector<u8> {
+             b"Hello
+             Libra"
+    }
+}
```

Comment:
Could you add similar tests in `language/move-lang/tests/functional/` to test the expected byte outputs (with runtime asserts)

Reply:
Also, could you add tests for other escape sequences?

Reply:
Will do.

Reply:
Done.